app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -1121,6 +1122,35 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPassword_clickforgotPin_enterAdminPin_clickSubmit_dialogMessageIsCorrect() {
+    ActivityScenario.launch<PinPasswordActivity>(
+      PinPasswordActivity.createPinPasswordActivityIntent(
+        context = context,
+        adminPin = adminPin,
+        profileId = userId
+      )
+    ).use {
+      onView(withId(R.id.pin_password_input_pin_edit_text))
+        .perform(editTextInputAction.appendText(""), closeSoftKeyboard())
```

Comment:
Is this step necessary in order to click the forgot pin button?

Reply:
Added this to maintain flow of the test.

Reply:
Sorry @pratyaksh1610, I'm not sure that I follow. Let me try to rephrase: why is this step needed for the test to succeed?

Reply:
@BenHenning I think this will not be needed as we need to test the forgot pin message and not the input text, so we can start from  `onView(withId(R.id.forgot_pin)).perform(click())` click on the forgot pin and the remaining remains the same.
We can remove it.

Reply:
That sounds reasonable--could you please go ahead and make that change?